app/views/mentor/students/_student.html.slim

```diff
@@ -11,8 +11,8 @@ div.panel.panel-default
   div.panel-body #{auto_link simple_format(student.learning_history)}
 
 div.panel.panel-default
-  div.panel-heading #{ApplicationDraft.human_attribute_name :student_application_code_background }
-  div.panel-body #{auto_link simple_format(student.code_background)}
+  div.panel-heading #{ApplicationDraft.human_attribute_name :student_application_language_learning_period }
+  div.panel-body #{auto_link simple_format(student.language_learning_period)}
```

Comment:
I'd spare the `auto_link` here. The field is not intended to contain html-links and I'd suggest to just keep things as simple as possible here.